inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_b_fs_zyx_fsv16_imad.cl

```diff
@@ -40,6 +55,12 @@ KERNEL(convolution_gpu_b_fs_zyx_fsv16_imad)(
     const __global FILTER_TYPE *weights,
 #if BIAS_TERM
     const __global BIAS_TYPE *biases,
+#endif
+    #if ASYMMETRIC_WEIGHTS_QUANTIZATION
+    const __global WEIGHTS_ZERO_POINTS_TYPE *weights_zp,
+#endif
+#if ASYMMETRIC_DATA_QUANTIZATION
+    const __global ACTIVATIONS_ZERO_POINTS_TYPE *activations_zp,
```

Comment:
one argument (compensation) is missing. How does it work? I suppose if this kernel is somehow selected, then you'll have setArg error

Reply:
Fixed